app/jobs/descriptive_metadata_export_job.rb

```diff
@@ -35,6 +38,11 @@ def perform(bulk_action_id, _params)
 
   private
 
+  def field_name(key)
+    field = key.split(':').first
```

Comment:
Colon is no longer used. It's a `.` now.

Reply:
`/[[:alpha:]]*/.match(key)[0]` might be a way to avoid that problem and combine the two lines into one.

Reply:
Oh, thanks, didn't come across [:alpha:] before. Will add that in.